src/apps/peer/mainudpserver.c

```diff
@@ -36,7 +36,11 @@
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
+#if defined(_MSC_VER)
```

Comment:
In `src/apps/natdiscovery/natdiscovery.c` this appears as `#if defined(WINDOWS)`
I *think* WINDOWS covers MSVC and MinGW but not vice-versa.

Reply:
Because of mingw has getopt.h. only msvc isn't it.